Sources/FluentKit/Properties/CompositeID.swift

```diff
@@ -74,11 +74,13 @@ extension CompositeIDProperty: AnyDatabaseProperty {
 
 extension CompositeIDProperty: AnyCodableProperty {
     public func encode(to encoder: Encoder) throws {
-        try self.value!.encode(to: encoder)
+        var container = encoder.singleValueContainer()
+        try container.encode(self.value!)
```

Comment:
I know this is probably safe but should we try a guard and throw an error in the case it should never hit?

Reply:
A `CompositeID` property with no value set will have already crashed out in 20 other places before it gets anywhere near here; the only reason the value is optional is the definition of `id` on `Model` absolutely requires it (despite the fact that it's not a supported scenario for composite ID models 🤦‍♀️).